README.md

```diff
@@ -433,6 +433,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Aysha Hakeem](https://github.com/AyshaHakeem)
   - [Ayush Chaudhary](https://github.com/YushChaudhary)
   - [Ayush Kumar](https://github.com/Ayush7614)
+  - [Ayush K.S.](https://github.com/ayushdotdev)
```

Comment:
AFAIK, this should be in line 435, after Ayush Chaudhary.
`S` comes first. Later on, the `U` digit.

Reply:
Hi @Panquesito7 what should I do now? 
Should I create a new branch for new pull request or if I make another commit from the current branch ayushdotdev will it be fine ?

Reply:
> Hi @Panquesito7 what should I do now? Should I create a new branch for new pull request or if I make another commit from the current branch ayushdotdev will it be fine ?

Hi, it's ok if you commit from this branch, it'll update this pull request.

Reply:
Thanks for your help @mkubdev 😁